_analyzers/token-filters/delimited-payload.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: default
+title: Delimited payload
+parent: Token filters
+nav_order: 90
+---
+
+# Delimited payload token filter
+
+The `delimited_payload` token filter is used to parse tokens containing payloads during the analysis process. For example, the string `red|1.5 fast|2.0 car|1.0` is parsed into tokens `red` (with a payload of `1.5`), `fast` (with a payload of `2.0), and `car` (with a payload of `1.0`). This is particularly useful when your tokens have additional associated data (like weights, scores, or other numeric values) that you can use for scoring or custom query logic. The filter can handle different types of payloads, including integer, float, and strings. and attach payloads (extra metadata) to tokens
+
```

Comment:
Above: Confirm that my changes to the last sentence retain technical accuracy.

Reply:
This still makes sense